### PR TITLE
Add MSI installation to documentation

### PR DESCRIPTION
**Pull request by russcam**
Move installation documentation for Windows with the .zip archive into the zip and tar installation documentation, and clearly indicate any differences for installing on macOS/Linux and Windows.

### CODE REVIEW

File: docs/reference/setup/install/windows.asciidoc [hunk 1]
```diff
@@ -25,236 +28,408 @@ endif::[]
...
+[[msi-command-line-options]]
+==== Command line options
 
-The commands available are:
+All settings exposed within the GUI are also available as command line arguments (referred to
+as _properties_ within Windows Installer documentation) that can be passed to msiexec:
 
 [horizontal]
-`install`:: Install Elasticsearch as a service
+`INSTALLDIR`::
```
Comment (clintongormley) [inline docs/reference/setup/install/windows.asciidoc]: This seems to be a repetition of a section above.  Could these two not be combined?

**Review by clintongormley — COMMENTED** [review]
Wow - quite a PR! :)

I didn't read all the instructions in detail - I trust you know what you're talking about.

One thought - combining the windows zip instructions with zip-targz makes the page rather long and complicated.  Might be easier to keep a separate zip-on-windows page?

File: docs/reference/setup/install/windows.asciidoc (hunk 1 above)
Comment (russcam) [inline docs/reference/setup/install/windows.asciidoc]: This section serves as the canonical set of properties that the installer accepts (case-sensitive) so I'll look to remove the repeated information above

**Comment by russcam** [comment]
Thanks for taking a look @clintongormley!

> One thought - combining the windows zip instructions with zip-targz makes the page rather long and complicated. Might be easier to keep a separate zip-on-windows page?

Was thinking the same too, the waters are a little bit muddied in the single file 😄 I'll update to separate

**Comment by russcam** [comment]
I've updated @clintongormley, if you wouldn't mind taking another look. I've split out the windows zip installation instructions into a separate page

**Review by clintongormley — APPROVED** [review]

**Comment by clintongormley** [comment]
thanks @russcam

**Comment by russcam** [comment]
@clintongormley should this be ported to `5.x` and `5.5`?

**Pull request merged**